src/btree/bt_page.c

```diff
@@ -418,20 +454,178 @@ __wt_page_inmem(WT_SESSION_IMPL *session, WT_REF *ref, const void *image, uint32
     return (ret);
 }
 
+/*
+ * __wt_col_fix_read_auxheader --
+ *     Read the auxiliary header following the bitmap data, if any. This code is used by verify and
+ *     needs to be accordingly careful. It is also used by mainline reads so it must also not crash
+ *     or print.
+ */
+int
+__wt_col_fix_read_auxheader(
+  WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_COL_FIX_AUXILIARY_HEADER *auxhdr)
+{
+    WT_BTREE *btree;
+    uint32_t bitmapsize, auxheaderoffset;
+    const uint8_t *raw, *end;
```

Comment:
fixed

Reply:
Is the fix missed? `end` should be first and then `raw` right?

Reply:
Hmm. Not sure how that happened. Really fixed now... but not yet pushed, since something failed and I don't want to erase that until someone looks at it (and can't see it myself). Will be in the next set of pushes, all queued up.

Reply:
```[2021/11/03 05:44:04.875] Running test command: /data/mci/f37526e8c0a2783719916f5bf77f9c27/wiredtiger/build_posix/test/csuite/test_wt4803_history_store_abort
[2021/11/03 05:44:14.656] [1635918254:656767][46559:0x7f27afbfe700], file:test_wt4803_history_store_abort.wt, eviction-server: int __wt_hs_insert_updates(WT_SESSION_IMPL *, WT_RECONCILE *, WT_MULTI *), 755: WiredTigerHS: file size of 105271296 exceeds maximum size 104857600: WT_PANIC: WiredTiger library panic: WT_PANIC: WiredTiger library panic
[2021/11/03 05:44:14.656] Got history store error (expect_panic=true)
[2021/11/03 05:44:14.658] [1635918254:658206][46559:0x7f27b3e57cc0], file:test_wt4803_history_store_abort.wt, WT_CURSOR.update: int __wt_hs_insert_updates(WT_SESSION_IMPL *, WT_RECONCILE *, WT_MULTI *), 755: WiredTigerHS: file size of 105271296 exceeds maximum size 104857600: WT_PANIC: WiredTiger library panic: WT_PANIC: WiredTiger library panic
[2021/11/03 05:44:14.658] Got history store error (expect_panic=true)
```

Is the above failure is referring to? I saw this failure and I am not sure whether it has any relation to PR.

Reply:
Keith said it didn't, so I guess not...